packages/@aws-cdk/aws-s3-deployment/lib/bucket-deployment.ts

```diff
@@ -0,0 +1,64 @@
+import cloudformation = require('@aws-cdk/aws-cloudformation');
+import lambda = require('@aws-cdk/aws-lambda');
+import s3 = require('@aws-cdk/aws-s3');
+import cdk = require('@aws-cdk/cdk');
+import { ISource } from './source';
+
+const handlerCodeBundle = require.resolve('@aws-cdk/aws-s3-deployment-handler/dist/lambda.zip');
```

Comment:
Why is the handler code not in the same package? Does this appeal to an inner sense of tidiness? I really don't see the advantage of spreading this over two packages.

Reply:
I guess you are right. Let me fix.